docs/en/ingest-management/fleet-limitations.asciidoc

```diff
@@ -20,16 +20,15 @@ This feature has additional limitations at the current time:
 *   Requires internet access for {kib} to download integration packages
```

Comment:
For line 19, I thought that we added proxy support. The What's new for 7.10 says: "Configure a proxy in Fleet to access the internet." We haven't documented that yet, either.

Reply:
I will do a followup for the proxy support with @jfsiii